src/Parser/Rogue/Subtlety/Modules/Talents/DarkShadow/DarkShadowEvis.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import DarkShadow from './DarkShadow';
+
+
+class DarkShadowEvis extends DarkShadow {
+
+  statistic() {    
+  const totalEviscerateHitsInShadowDance  = this.danceDamageTracker.getAbility(SPELLS.EVISCERATE.id).damageHits;
+    const danceEvis = totalEviscerateHitsInShadowDance / this.totalShadowDanceCast;
+    return (
+        <StatisticBox
+          icon={<SpellIcon id={SPELLS.EVISCERATE.id} />}
+          value={`${danceEvis.toFixed(2)}`}
+          label="Average Eviscerates in Shadow Dance"
+          tooltip={`Your average Eviscerate casts per Shadow Dance. Your cast ${totalEviscerateHitsInShadowDance} Eviscerates in ${this.totalShadowDanceCast} Shadow Dances. This number includes Eviscerates cast from Death from Above. Subtlety rogue should cast as many as possible (usually 2 times) Eviscerates in a Shadow Dance to get benefit from 30% damage increasing of Dark Shadow talent.`}
+          />
```

Comment:
If you have a inclusion of "Subtlety rogue should cast as many as possible (usually 2 times) Eviscerates in a Shadow Dance to get benefit from 30% damage increasing of Dark Shadow talent." in the tooltip, then it seems clear that this should instead be a suggestion for this specific module.

Reply:
This was an old statistic in this module before I picked it up. I'm still on the fence about it, its defiantly not a suggestion - the proper suggestions have been created in this PR (for example cast 4 spells during dance). However it can be a useful statistic just like "Dark Shadow damage contribution"  to see why you are performing worse then someone else. I was planing to get some extra opinions on it and then decide to remove or not.

Reply:
It should 100% be a suggestion and not in the tooltip.

Reply:
But its not a valid suggestion. Its sometimes a mistake to cast 2 Eviscerates in  Dance. I can just remove it now if it helps

Reply:
Just not sure why it has to be handled right now considering its old and unrelated to this PR

Reply:
Because you're adjusting it in a PR, you might as well fix it properly. If it's majorly handled by something newly added, just remove it or reword it to correctly showcase what you want it to showcase. 
This could be something like ``You should cast as many Eviscerates as necessary to get four total spell casts in Shadow Dance``.

Reply:
Ok I've added suggestion because I want to keep it in for now so I can get some feedback on it.